### PR TITLE
[ci] gh: new, 2.1.0

### PR DESCRIPTION
**Pull request by KexyBiscuit**
Topic Description
-----------------

Introduce `gh`.

Package(s) Affected
-------------------

`gh`

Security Update?
----------------

No

Test Build(s) Done
------------------

**Primary Architectures**

- [x] AMD64 `amd64`   
- [x] AArch64 `arm64`
    

**Secondary Architectures**

Architectural progress for "secondary," or experimental ports does not impede on merging of this topic.

- [x] Loongson 3 `loongson3`
- [x] PowerPC 64-bit (Little Endian) `ppc64el`

Update(s) Uploaded to Stable
----------------------------

**Primary Architectures**

- [x] AMD64 `amd64`   
- [x] AArch64 `arm64`

**Secondary Architectures**

Architectural progress for "secondary," or experimental ports does not impede on merging of this topic.

- [x] Loongson 3 `loongson3`
- [x] PowerPC 64-bit (Little Endian) `ppc64el`

### CODE REVIEW

**Comment by KexyBiscuit** [comment]
cli/cli#4430

**Comment by KexyBiscuit** [comment]
Smoke test passed.

File: extra-vcs/gh/autobuild/defines [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+PKGSEC=vcs
+PKGDES="GitHub’s official command line tool"
+
+PKGDEP="git"
+BUILDDEP="go"
+
+ABTYPE=plainmake
+MAKE_AFTER="CGO_ENABLED=0"
+
+ABSPLITDBG=0
```
Comment (WhiredPlanck) [inline extra-vcs/gh/autobuild/defines:11]: Why not split debug package ?

Comment (WhiredPlanck) [reply extra-vcs/gh/autobuild/defines:11]: For packages solely consisting of rust or go programs, it's unnecessary to split `-dbg` package. So never mind this.

**Comment by KexyBiscuit** [comment]
Smoke test passed for 2.1.0.

**Review by MingcongBai — APPROVED** [review]

**Pull request merged**